libs/postgres_backend/src/lib.rs

```diff
@@ -288,6 +290,25 @@ impl PostgresBackend {
             peer_addr,
         })
     }
+}
+
+impl<IO: AsyncRead + AsyncWrite + Unpin> PostgresBackend<IO> {
+    pub fn new_from_io(
```

Comment:
it can be merged with `fn new()`

Reply:
Do you mean call `new_from_io()` inside `new()`, or remove `new()` altogether (then we will need to always pass peer_addr)?

Reply:
Ah, I forgot about peer_addr. Then either variant is ok, including current.